router/http/src/main/scala/io/buoyant/router/http/Identifier.scala

```diff
@@ -5,6 +5,14 @@ import com.twitter.finagle.buoyant.Dst
 import com.twitter.util.Future
 import io.buoyant.router.RoutingFactory
 
+object Identifier {
+  def mk(
+    prefix: Path,
+    uris: Boolean = false,
+    baseDtab: () => Dtab = () => Dtab.base
+  ): RoutingFactory.Identifier[http.Request] = Identifier(prefix, uris, baseDtab)
+}
+
 case class Identifier(
```

Comment:
Should this move into the identifier/http project?  It's used as a default value for the identifier param so I guess not.